ext/standard/var.c

```diff
@@ -929,11 +916,10 @@ static void php_var_serialize_nested_data(smart_str *buf, zval *struc, HashTable
 static void php_var_serialize_class(smart_str *buf, zval *struc, zval *retval_ptr, php_serialize_data_t var_hash) /* {{{ */
 {
 	HashTable props;
-	if (php_var_serialize_get_sleep_props(&props, struc, HASH_OF(retval_ptr)) == SUCCESS) {
-		php_var_serialize_class_name(buf, struc);
-		php_var_serialize_nested_data(
-			buf, struc, &props, zend_hash_num_elements(&props), /* incomplete_class */ 0, var_hash);
-	}
+	php_var_serialize_get_sleep_props(&props, struc, HASH_OF(retval_ptr));
```

Comment:
Similarly, because serializing can throw an exception (in __serialize, emitting notices, etc), the old code should be kept

Reply:
(same comment as above)